mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -3578,7 +3561,9 @@ ParseResult MatmulOp::parse(OpAsmParser &parser, OperationState &result) {
   }
   // Initialize indexingMaps, if not supplied explicitly.
   if (indexingMapsAttr.empty()) {
-    indexingMapsAttr = getDefaultMatmulIndexingMapAttr(result.getContext());
+    indexingMapsAttr = llvm::map_to_vector(
+        MatmulOp::getDefaultIndexingMaps(parser.getContext()),
+        [](AffineMap map) -> Attribute { return AffineMapAttr::get(map); });
   }
```

Comment:
This is because user may choose not to explicitly provide the indexing map as it is optional. In this case compiler has to consider the default map and update `indexing_mps `attribute accordingly.